src/coreclr/jit/importercalls.cpp

```diff
@@ -3398,161 +3398,86 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                 retNode = impMathIntrinsic(method, sig, callType, ni, tailCall);
                 break;
             }
-#if defined(TARGET_ARM64)
-            // ARM64 has fmax/fmin which are IEEE754:2019 minimum/maximum compatible
-            case NI_System_Math_Max:
-            case NI_System_Math_Min:
-#endif
 
-#if defined(FEATURE_HW_INTRINSICS) && defined(TARGET_XARCH)
             case NI_System_Math_Max:
-            case NI_System_Math_Min:
```

Comment:
This logic was all extracted to a new helper function `impMinMaxIntrinsic`, to avoid code duplication.

Reply:
Doing so made the diff unhappy, and so it's showing a bunch of code as "new" where it actually isn't